### PR TITLE
[MISC] Remove 'using namespace seqan3' from test/unit/*.

### LINKED ISSUE #1576: [MISC] Remove 'using namespace seqan3' from test/unit/* ... more than one issue (207 files)
Remove 'using namespace seqan3' from
- [x] test/unit/alignment
- [x] test/unit/alphabet
- [x] test/unit/argument parser
- [x] test/unit/contrib
- [x] test/unit/core
- [x] test/unit/io
- [ ] test/unit/range
- [x] test/unit/search
- [x] test/unit/std

### PR DESCRIPTION
**Pull request by Irallia**
Last cleanup for test/unit/*.
Resolves #1576

### CODE REVIEW

File: test/unit/alignment/matrix/detail/trace_iterator_test.cpp [hunk 1]
```diff
@@ -7,17 +7,19 @@
...
 
 #include <seqan3/alignment/matrix/detail/trace_iterator.hpp>
 #include <seqan3/alignment/matrix/detail/two_dimensional_matrix.hpp>
 #include <seqan3/alignment/matrix/trace_directions.hpp>
 #include <seqan3/range/views/to.hpp>
-#include <seqan3/std/iterator>
 
 #include "../../../range/iterator_test_template.hpp"
 
+using seqan3::operator|;
```
Comment (simonsasse) [inline test/unit/alignment/matrix/detail/trace_iterator_test.cpp:21]: why did the "|" Operator work before you changed this file ? Or, if it did work before, why did you add it here ?

Comment (Irallia) [reply test/unit/alignment/matrix/detail/trace_iterator_test.cpp:21]: hope I can explain this correctly:
It's an issue with the enum_bitwise_operators, that the ADL is not working and so the operators are not visible. We discussed this in another PR: https://github.com/seqan/seqan3/pull/1565 and added a discussion Card: https://github.com/orgs/seqan/projects/4#card-32890520

Comment (simonsasse) [reply test/unit/alignment/matrix/detail/trace_iterator_test.cpp:21]: thanks !:)
